javascript/packages/orchestrator/src/networkNode.ts

```diff
@@ -97,15 +97,17 @@ export class NetworkNode implements NetworkNodeInterface {
   async pause() {
     const client = getClient();
     const args = client.getPauseArgs(this.name);
+    const scoped = client.providerName === "kubernetes";
 
-    const result = await client.runCommand(args, { scoped: true });
+    const result = await client.runCommand(args, { scoped });
     return result.exitCode === 0;
   }
 
   async resume() {
     const client = getClient();
     const args = client.getResumeArgs(this.name);
-    const result = await client.runCommand(args, { scoped: true });
+    const scoped = client.providerName === "kubernetes";
+    const result = await client.runCommand(args, { scoped });
```

Comment:
can this repeatable code (with the `const client = getClient();` become reusable in the class maybe?

Reply:
At the moment, `client` is a `singleton` we can refactor this to have get the client in the class constructor and then use `this.client`. But I think is not in the scope of this `pr`.